dotCMS/src/main/java/com/dotcms/rest/exception/mapper/ElasticsearchStatusExceptionMapper.java

```diff
@@ -0,0 +1,29 @@
+package com.dotcms.rest.exception.mapper;
+
+import com.dotmarketing.util.Logger;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import org.elasticsearch.ElasticsearchStatusException;
+import javax.ws.rs.ext.ExceptionMapper;
+
+public class ElasticsearchStatusExceptionMapper implements ExceptionMapper<ElasticsearchStatusException> {
+
+
+    @Override
+    public Response toResponse(ElasticsearchStatusException exception) {
+
+        //Log into our logs first.
+        Logger.warn(this.getClass(), exception.getMessage(), exception);
+
+        //Create the message.
+        String message = exception.getMessage();
+
+        //Creating the message in JSON format.
+        String entity = ExceptionMapperUtil.getJsonErrorAsString(message);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'entity' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5171086)